frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/top-consumers-card/utils.ts

```diff
@@ -19,6 +19,25 @@ export const sortResources: SortResourcesProps = (a, b) => {
   const bVal = _.get(b, 'values');
   const x = _.get(a, ['values', aVal.length - 1, 1]);
   const y = _.get(b, ['values', bVal.length - 1, 1]);
+  if (x === y) {
+    const aNameVal = _.get(a, 'metric');
+    const bNameVal = _.get(b, 'metric');
+    if (aNameVal.storageclass) {
+      return aNameVal.storageclass < bNameVal.storageclass
+        ? -1
+        : aNameVal.storageclass > bNameVal.storageclass
+        ? 1
+        : 0;
```

Comment:
Looks repetitive. create a function and pass` aNameVal, BNameVal and metricType`